tests/ddd-deriv/input.dat

```diff
@@ -17,31 +17,33 @@ molecule h2o {
 set basis cc-pVDZ
 ans = -76.0266327341067125
 
-energy('scf')
+largs = {"protocols": {"stdout": False}, "tag": "test1"}
```

Comment:
Are the `protocols` and `tag` kwarg documented anywhere? I'm not sure what they do.

Reply:
Docs aren't great. `protocols` is here https://molssi.github.io/QCElemental/api/qcelemental.models.AtomicInput.html#qcelemental.models.AtomicInput.protocols , but it looks like I need to add that class to the autodoc list for expanded documentation. Basically, protocols doesn't change what gets computed, but it's a way for the user to change the layout/contents of what gets returned -- don't want miles of text, turn off `stdout` or do want every stage of the optimization, switch from `first_and_last` to `all`.

`tag` is in the qcf domain, so I'm not so familiar with finding its docstring. Generally have to isolate the `next` branch and look it up https://github.com/MolSSI/QCFractal/blob/next/qcportal/qcportal/client.py#L903 . What it means is that you want to pass a string "tag" to the distributed job. Then in the qcf config, if you've labeled a series of compute nodes (with say, local scratch or high memory) with that same tag, qcf knows to send these jobs there. Useful when you've got heterogeneous projects or types of calcs running on a cluster and you want to direct them as you could with slurm.